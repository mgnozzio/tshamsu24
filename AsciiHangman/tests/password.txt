pAsSWorD
r
s
t
l
n
e
p
o
a
d
w
