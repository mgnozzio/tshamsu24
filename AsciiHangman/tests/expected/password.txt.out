Welcome to the ASCII Version of Hangman!
For best results, please run this from the command line.
Please enter a secret word: 
















































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































   ____  
  |    | 
  |      
  |      
  |      
__|__    
Puzzle to solve: _ _ _ _ _ _ _ _ 
Please guess a letter: R
Puzzle to solve: _ _ _ _ _ _ R _ 
Please guess a letter: S
Puzzle to solve: _ _ S S _ _ R _ 
Please guess a letter: T

   ____  
  |    | 
  |    O 
  |      
  |      
__|__    
Puzzle to solve: _ _ S S _ _ R _ 
Please guess a letter: L

   ____  
  |    | 
  |    O 
  |    | 
  |      
__|__    
Puzzle to solve: _ _ S S _ _ R _ 
Please guess a letter: N

   ____  
  |    | 
  |    O 
  |   \| 
  |      
__|__    
Puzzle to solve: _ _ S S _ _ R _ 
Please guess a letter: E

   ____  
  |    | 
  |    O 
  |   \|/
  |      
__|__    
Puzzle to solve: _ _ S S _ _ R _ 
Please guess a letter: P
Puzzle to solve: P _ S S _ _ R _ 
Please guess a letter: O
Puzzle to solve: P _ S S _ O R _ 
Please guess a letter: A
Puzzle to solve: P A S S _ O R _ 
Please guess a letter: D
Puzzle to solve: P A S S _ O R D 
Please guess a letter: W
Success!  Player 2 wins!
